src/components/DriverCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+function DriverCard(props) {
+  const { name, rating, img, car } = props;
+
+  return (
+    <div>
+      <h2>{name}</h2>
+
+      {Math.floor(rating)}
+      {rating < 1 && rating >= 0 && <h2>☆☆☆☆☆</h2>}
+      {rating === 1 && <h2>★☆☆☆☆</h2>}
+
+      {rating < 2 && rating >= 1 && <h2>★☆☆☆☆</h2>}
+      {rating === 2 && <h2>★★☆☆☆</h2>}
+
+      {rating < 3 && rating >= 2 && <h2>★★☆☆☆</h2>}
+      {rating === 3 && <h2>★★★☆☆</h2>}
+
+      {rating < 4 && rating >= 3 && <h2>★★★☆☆</h2>}
+      {rating === 4 && <h2>★★★★☆</h2>}
+
+      {rating < 5 && rating >= 4 && <h2>★★★★☆</h2>}
+      {rating >= 5 && <h2>★★★★★</h2>}
```

Comment:
You already have a Component that takes the number from props.children and converts it into stars it's the Rating Component. 

So instead of having to do the conditions for the stars again, you can just import the Rating component at the top of the page, and call it inside of the function with rating as children:
```
import React from 'react';
import Rating from './Rating'

function DriverCard(props) {
  const { name, rating, img, car } = props;

  return (
    <div>
      <h2>{name}</h2>

      <Rating>{rating}<Rating/>

      <img src={img} alt="Profile Image" />
      <h3>{car.model}</h3>
      <h3>{car.licensePlate}</h3>
    </div>
  );
}
export default DriverCard;
```